packages/react/src/SubNav/SubNav.module.css

```
@@ -232,7 +241,7 @@
   .SubNav__link-label::after {
     content: '';
     position: absolute;
-    bottom: -6px;
+    bottom: -9px;
```

Can we go with -8px here instead? We can then use the base token and bring the line a little bit closer (feels quite far right now)


something like...

```suggestion
    bottom: calc(var(--base-size-8) * -1);
```